ci/README.md

```diff
@@ -0,0 +1,89 @@
+# Theia CI
+
+This directory includes all the scripts required to run CI on Theia.
+
+For information about our Jenkins CI jobs and how to run the same tests locally,
+see [here](jenkins/README.md).
+
+For K8s conformance upstream tests, the version of the K8s conformance container
```

Comment:
Again, it does not make sense to repeat Antrea contents. Could we just talk about anything different in Theia, and point to the Antrea doc for shared ones?

Reply:
Thanks for reviewing. You're right. I've pointed the docs to Antrea doc and only listed the differences between Theia and Antrea.